synapse/lib/layer.py

```diff
@@ -162,26 +164,26 @@ class Layer(s_base.Base):
             await func(oper)
 
         if prov is None:
-            alias = None
+            iden = None
         elif isinstance(prov, bytes):
-            alias = prov
+            iden = prov
         else:
-            alias = await self._storProvStack(prov)
+            iden = await self._storProvStack(prov)
 
         if splices is not None:
-            if alias is not None:
+            if iden is not None:
                 for splice in splices:
-                    splice[1]['prov'] = alias
+                    splice[1]['prov'] = s_common.ehex(iden)
```

Comment:
compute and store this result for use once prior to entering the loop, instead of doing it inside of the loop

Reply:
roger